harness/propertyHelper.js

```diff
@@ -73,6 +73,18 @@ function verifyProperty(obj, name, desc, options) {
     }
   }
 
+  if (Object.prototype.hasOwnProperty.call(desc, 'get')) {
```

Comment:
This one is not quite useful.

There are ~30 accessors in spec.
All of them are non-enumerable, configurable, and lack `set` (well, except for `Object#__proto__`).
We can introduce a new helper to check their descriptors:

```js
verifyGetterProperty(Map.prototype, 'size');
```

Reply:
I'd prefer deferring this change to when we have usage and tests for it. It's pretty tricky to check/compare the value of the get functions and we might end up not having a proper expected value.

Reply:
6935757